api/v1alpha1/k6_types.go

```diff
@@ -38,6 +38,17 @@ type Pod struct {
 	ServiceAccountName           string                        `json:"serviceAccountName,omitempty"`
 	SecurityContext              corev1.PodSecurityContext     `json:"securityContext,omitempty"`
 	EnvFrom                      []corev1.EnvFromSource        `json:"envFrom,omitempty"`
+	InitContainers               []InitContainer               `json:"initContainers"`
+}
+
+type InitContainer struct {
+	Image           string                 `json:"image,omitempty"`
```

Comment:
I don't think `Image` can be `omitempty`. The docs seem to suggest use cases when one needs it empty [here](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#image) but if try to specify init container without image, I'll get this error in runtime:
```
Failed to launch k6 test initializer    {"namespace": "default", "name": "k6-sample-init-container", "error": "Job.batch \"k6-sample-init-container-initializer\" is invalid: spec.template.spec.initContainers[1].image: Required value"}
```
If we remove `omitempty` then this kind of error should be caught at validation step.